packages/tempus/test/BackwardEuler/Tempus_BackwardEuler_ASA.cpp

```diff
@@ -64,9 +64,12 @@ TEUCHOS_UNIT_TEST(BackwardEuler, SinCos_ASA)
       getParametersFromXmlFile("Tempus_BackwardEuler_SinCos.xml");
```

Comment:
Should the explicit case be tested too?  I am not requiring to do it in Tempus - I could try it in Piro, since I will be adding tests there anyway.

Reply:
See above.  It is tested in one case using an explicit adjoint ME for the SinCos model.  What I cannot test is the Tpetra adapter, because Tempus has no Tpetra tests.  I think testing it in Piro would be the best place.

Reply:
Ok, yes, makes sense.